docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,16 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
+selectively "pausing" functionality of specific modules on a running network.
+The circuit breaker is intended to be enabled through either:
+ - governance, 
+ - the bonded validator group (for emergencies), 
```

Comment:
Should we have commas at the end here?

Reply:
how is this different than the bonded validator set just stopping the consensus process?

Reply:
@alexanderbez it's how I learned it in engineering technical writting (to read as if there where no bullets) but I don't really care